components/mdx/Author.tsx

```diff
@@ -1,14 +1,73 @@
-import { Box, Text } from '@chakra-ui/react'
```

Comment:
Have you tried it, or would it be possible to display it without the box @briangershon ?  I would guess that it attracts enough attention with the question mark avatar and also the fact that the word 'forum' is underlined.  Also since the text is smaller (which is really nice) than the rest of the lesson text, it draws your eye to it as some sort of 'notice to read'.....Just a few musings

Reply:
Thanks @elPiablo sounds good, I'll push up changes to remove the border.

Here's the version before and after.

We can always tweak later if any other tweaks on this.

## Before

<img width="1232" alt="Screenshot 2023-04-12 at 6 41 07 PM" src="https://user-images.githubusercontent.com/139330/231624791-a8e615a7-52c8-486b-8d86-1c9f90f2694f.png">

## After

<img width="1187" alt="Screenshot 2023-04-12 at 6 46 48 PM" src="https://user-images.githubusercontent.com/139330/231625276-852b908b-0af9-4462-a134-e41f6e99aa06.png">